### ISSUE TITLE
undefined reference in disable_raw_mode()

### ISSUE THREAD

**Issue by remy-luisant** [kdheepak/TerminalUserInterfaces.jl#10]
This could be a case of me misusing the library, but enable_raw_mode() works just fine and lets me get the work done. It does not seem like I can disable it, however.

```
julia> TerminalUserInterfaces.disable_raw_mode()
ERROR: UndefRefError: access to undefined reference
Stacktrace:
 [1] getproperty
   @ ./Base.jl:33 [inlined]
 [2] getindex
   @ ./refvalue.jl:56 [inlined]
 [3] disable_raw_mode()
   @ TerminalUserInterfaces ~/.julia/packages/TerminalUserInterfaces/Mhqzn/src/utils.jl:191
 [4] top-level scope
   @ REPL[2]:1
```

**Comment by kdheepak** [comment]
I think this is a bug. Can you try calling `backup_termios()` before `enable_raw_mode`?

**Comment by remy-luisant** [comment]
This appears to fix the problem.

**Comment by kdheepak** [comment]
I’m guessing you aren’t calling `initialize`: https://github.com/kdheepak/TerminalUserInterfaces.jl/blob/9d55682a1c9ce67f2af292518f7b0cd3aa2efefd/src/utils.jl#L202

**Comment by remy-luisant** [comment]
No, I'm using just the one call to enable raw mode, since all I wanted to get is keyboard control. initialize() would clear the screen, hide the cursor, etc...

**Comment by kdheepak** [comment]
Got it. I’ll have to add a `__init__` to the package and call `backup_termios` there.
